appsignal.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |gem| # rubocop:disable Metrics/BlockLength
   gem.extensions            = %w[ext/extconf.rb]
 
   gem.metadata = {
+    "rubygems_mfa_required" => "true",
     "bug_tracker_uri"   => "https://github.com/appsignal/appsignal-ruby/issues",
```

Comment:
It seems like the other keys in this hash are aligned with each other with spaces. I personally don't mind, and RuboCop didn't seem to either, but just pointing it out in case you'd like to fix the alignment.

Reply:
Personally I don't want them to align. I don't like that in hashes. Updating them all to match means changing a bunch more lines 😐
I think RuboCop is set to ignore this scenario, along with a bunch of other things in the todo, until we have time to decide on it.

Reply:
Sounds good to me. Never been a fan of it either.